odoo/tools/translate.py

```diff
@@ -1125,7 +1137,7 @@ def _babel_extract_terms(self, fname, path, root, extract_method="python", trans
 
     def _export_translatable_resources(self):
         """ Export translations for static terms
-        
+
```

Comment:
Don't do that.

Reply:
Why not :thinking: ?

Reply:
because it touches bits completely unrelated to the actual patch. Reformatting / fixing the bits which the PR actually touches is fine, but it doesn't come anywhere near this.